big_tests/tests/mam_helper.erl

```diff
@@ -889,7 +898,7 @@ parse_messages(Messages) ->
     end.
 
 bootstrap_archive(Config) ->
-    random:seed(os:timestamp()),
+    rand:seed(os:timestamp()),
```

Comment:
According to doc http://erlang.org/doc/man/rand.html#seed-1 `rand:seed/1` doesn't take the timestamp as the argument. This call crashes so many test cases in mam_SUITE are automatically skipped.